src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+
+    private final double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((radius * radius * Math.PI) * 10.0) / 10.0;
```

Comment:
Avoid using magic numbers in your code. The number 10.0 seems to be a magic number for rounding purposes. It's better to declare it as a constant with a clear name indicating its purpose.